system/data/es/systems/core.json

```diff
@@ -0,0 +1,270 @@
+{
+    "name": "Core (Básico)",
```

Comment:
Since Condensed and Accelerated are simply "Condensado" and "Acelerado", I'd name this as "Básico" as well, without the brackets, for consistency.